main.js

```diff
@@ -1,123 +1,209 @@
-var titleField = document.querySelector('#submitted-title');
```

Comment:
Let's see if we can reassign these to classes like you suggested in the refactoring!